modules/core/src/lib/layer.js

```diff
@@ -864,6 +864,26 @@ export default class Layer extends Component {
     }
   }
 
+  updateAutoHighlight(info) {
```

Comment:
Isn't this the new "lifecycle method" that should be documented and overridden by subclasses?

Reply:
`updateAutoHighlight` is meant to be called by code outside of this class, and `_updateAutoHighlight` is meant to be overriden by sub classes.

I prefer not to document this until we have a consensus on the right naming.